index.md

```diff
@@ -74,7 +74,7 @@ We can help you with, for example:
 We provide consultancy and engineering in research projects in various formats:
 - **Consultancy** in your project is always possible. Typically, consultancy consists of one or two meetings where we discuss your project and offer technical advice free of charge.
 - We also offer **short-term engineering** support as a free service. During a period of a few months, you will have regular meetings with one or two Research Engineers, with the aim of developing a proof-of-concept or exploring a new method for your research.
-- **Long-term support** is possible and can be tailored to your project, provided that funding is available for our services. We can also help you write the technical aspect of your grant proposal, in which our support is included.
+- **Long-term support** is possible and can be tailored to your project, provided that funding is available for our services. We can also help you write the technical aspect of your [grant proposal](_pages/Long-term-support/index.md), in which our support is included.
```

Comment:
Again, it is not clear to what is linked. I would include a sentence like: "More information on funding opportunities and criteria can be found here."

Reply:
Agree. the page is not (only) about technical support for proposals. It is also not about funding, per se. I linked to the "Long-term-support", bullet-header, as the page explains how a researcher can arrange long term support for a project